lib/ramble/ramble/pipeline.py

```diff
@@ -575,7 +575,14 @@ def _execute(self):
         fs.mkdirp(aux_software_dir)
         aux_repo_conf = os.path.join(aux_software_dir, "repos.yaml")
 
-        repo_conf_defs = [("repos", "repos.yaml"), ("modifier_repos", "modifier_repos.yaml")]
+        repo_conf_defs = [
```

Comment:
Can we read these from repository.py instead of replicating them? (i.e. https://github.com/GoogleCloudPlatform/ramble/blob/develop/lib/ramble/ramble/repository.py#L82 + `'.yaml'`

Reply:
Good call, done.